packages/client/hmi-client/src/components/workflow/tera-model-node.vue

```diff
@@ -52,15 +52,16 @@ const parameterValues = ref<StringValueMap>({});
 
 function run() {
 	if (ModelOperation.action) {
-		console.log(
-			ModelOperation.action({
-				model: model.value,
-				initialValues: initialValues.value,
-				parameterValues: parameterValues.value
-			})
-		);
+		// console.log(
+		// 	ModelOperation.action({
+		// 		model: model.value,
+		// 		initialValues: initialValues.value,
+		// 		parameterValues: parameterValues.value
+		// 	})
+		// );
```

Comment:
You can remove all console.log when doing PRs

Reply:
done